digital_land/cli.py

```diff
@@ -96,6 +96,31 @@ def collect_cmd(endpoint_path):
     collector.collect(endpoint_path)
 
 
+#
+#  collection commands
+#
+@cli.command(
+    "index",
+    short_help="create collection indices",
+)
+def index_cmd():
+    # TBD: replace with Collection()
+    indexer = Indexer()
+    indexer.index()
+
+
+@cli.command("collection-list-resources", short_help="list resources for a pipeline")
+def pipeline_resources_cmd():
+    collection = Collection()
+    # can be loaded from a collection directory, or a collection datapackage
+    collection.load()
```

Comment:
Can we call `load` from the constructor of `Collection`? Feels like it's part of initialising the object.